package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "github": "0.2.4",
-    "lambda-cfn": "0.1.4",
+    "lambda-cfn": "mapbox/lambda-cfn#update",
```

Comment:
Hi @alulsh , just changed it to use version 0.1.5 of lambda-cfn and works as expected. Thanks : )